x-pack/plugins/security_solution/public/common/lib/triggers_actions_ui/register_alerts_table_configuration.tsx

```diff
@@ -93,13 +93,28 @@ const registerAlertsTableConfiguration = (
     id: ALERTS_TABLE_REGISTRY_CONFIG_IDS.CASE,
     cases: { featureId: CASES_FEATURE_ID, owner: [APP_ID], syncAlerts: true },
     columns: alertColumns,
+
     getRenderCellValue: renderCellValueHookCasePage,
     useInternalFlyout,
     useBulkActions: getBulkActionHook(TableId.alertsOnCasePage),
     useCellActions: getUseCellActionsHook(TableId.alertsOnCasePage),
     sort,
     showInspectButton: true,
   });
+
+  registerIfNotAlready(registry, {
```

Comment:
fyi, I'm not going to block this going forward, but I did want you to know that there are some known performance items with the alert table, that users of the risk score may run into and you should be aware of https://github.com/elastic/security-team/issues/7217